api-guidelines/rest/compatibility/versioning-of-incompatible-changes/rules/must-use-absolute-profile-uris.md

```diff
@@ -6,11 +6,14 @@ id: R100066
 
 Profiles must be identified by absolute URIs (as defined in [RFC 6906](https://www.rfc-editor.org/rfc/rfc6906#section-3)) and documented in the OpenAPI specification.
 
-The URI identifying a profile must be treated as an identifier that should not change.
+The URI that identifies a profile must be treated as an identifier that should not change.
 While encouraged, it does not need to be resolvable.
 If it is resolvable, it should provide human-readable documentation of the profile.
 
-The URI should be in the same URL namespace as the APIs endpoints.For example, if all API endpoints are located at `https://api.otto.de/payment/`, the profile URIs should als be located at the same context path (e.g., `https://api.otto.de/payment/profiles/payment-method+v1`).
+Best practice:
```

Comment:
I would rather use `should` instead of best practice as `should` is an aux verb that is used to describe best practices throughout the guidelines (e.g., a `SHOULD` rule is a best practice).

Reply:
@thake Thanks for looking into this. As we use should as aux verb very prominently in the titles already I find "Best practice" the better structural element on editorial level.
Is it a showstopper for you if I keep it?